stem/solver.py

```diff
@@ -568,7 +599,7 @@ class SolverSettings:
     convergence_criteria: ConvergenceCriteriaABC
     reset_displacements: bool = False
     calculate_stresses_on_nodes: bool = True
-    strategy_type: StrategyTypeABC = field(default_factory=NewtonRaphsonStrategy)
+    strategy_type: StrategyTypeABC = field(default_factory=LinearNewtonRaphsonStrategy)
```

Comment:
i would keep the default as it was, as LinearNewtonRaphsonStrategy does not work in all cases

Reply:
but in all DYNAMIC benchmarks, where the strategy_type is not explicitly given, explicitly set LinearNewtonRaphsonStrategy